frameworks/Dart/aqueduct/server.dart

```diff
@@ -0,0 +1,228 @@
+import 'dart:async';
+import 'dart:io';
+import 'dart:math';
+
+import 'package:aqueduct/aqueduct.dart';
+import 'package:mustache/mustache.dart' as mustache;
+
+export 'dart:async';
+export 'dart:io';
+export 'package:aqueduct/aqueduct.dart';
+
+final _stripped_text = new ContentType("text", "plain");
+final _stripped_json = new ContentType("application", "json");
+
+final _random = new Random();
+
+// World table size
+const int _world_table_size = 10000;
+// Fortune table size used only for generation of data
+const int _FORTUNE_TABLE_SIZE = 100;
+
+const int _min_query_count = 1;
+const int _max_query_count = 500;
+
+Future main() async {
+  try {
+    var app = new Application<DartAqueductBenchmarkSink>();
+    var config = new ApplicationConfiguration()
+      ..port = 8080
+      ..configurationFilePath = "config.yaml";
+
+    app.configuration = config;
+    await app.start(numberOfInstances: 3, consoleLogging: false);
```

Comment:
It may make sense to use Platform.numberOfProcessors - 1 for number of instances. That seems to be the optimal amount.

Reply:
Ill set it so that if `numberOfProcessors` is more that 1 it'll be `Platform.numberOfProcessors - 1` otherwise set to `1`